area/area.go

```diff
@@ -135,7 +135,9 @@ func (m *GormAreaRepository) CheckExists(ctx context.Context, id uuid.UUID) erro
 func (m *GormAreaRepository) LoadMultiple(ctx context.Context, ids []uuid.UUID) ([]Area, error) {
 	defer goa.MeasureSince([]string{"goa", "db", "Area", "getmultiple"}, time.Now())
 	var objs []Area
-
+	if len(ids) == 0 {
```

Comment:
Add test case

Reply:
added in https://github.com/fabric8-services/fabric8-wit/pull/1865/commits/976f059d14c36e744ce3ee6a11b4203d0b77a119#diff-c29bce94e3f2014a8097c28266ddf430R297